README.rst

```diff
@@ -47,24 +47,32 @@ Here's an example
 .. code-block:: python
 
     # mycode.py
+    import procrastinate
 
     # Make an app in your code
-    app = procrastinate.App(connector=procrastinate.AiopgConnector())
+    app = procrastinate.App(connector=procrastinate.AiopgConnector(host="localhost", user="postgres", password="password"))
 
     # Open the connection to the database
-    app.open()
+    with app.open():
```

Comment:
I think we wanted to keep the sync example as simple as possible, and the context manager is not mandatory in sync mode. But I'm totally OK leaving that if you think this helps understanding better how it works.

Reply:
I suggest to leave it, as context managers have been "mainstream" for a long time, thus this code should still be pretty straight forward to understand. And it has the added benefit that all context managers have: Controlled resource setup and tear down.

Reply:
I am fine with using `with app.open()` in the snippets, but I think the `sum` task function should be declared outside the context.